src/Aquifer.API/Endpoints/Resources/Content/Get/Endpoint.cs

```diff
@@ -64,7 +65,9 @@ public override async Task HandleAsync(Request request, CancellationToken ct)
                     ParentResourceName = ar.ParentResource.DisplayName,
                     MediaTypes = ar.ResourceContents.Select(arrc => arrc.MediaType)
                 }),
-                ProjectEntity = rc.Projects.FirstOrDefault(),
+                ProjectEntity = rc.ProjectResourceContents.FirstOrDefault(prc => prc.ResourceContentId == request.Id) == null
+                    ? null
+                    : rc.ProjectResourceContents.First(prc => prc.ResourceContentId == request.Id).Project,
```

Comment:
This is the change that helped the most with performance by adding `WHERE [p1].[ResourceContentId] = @__request_Id_0` to the `ProjectResourceContents` sub-query.
SQL before:
```
DECLARE @__request_Id_0 INT = 6779;
SELECT [t].[EnglishLabel], [t].[DisplayName], [t].[c], [t].[LicenseInfo], [t].[Id], [t].[ResourceId], [t].[LanguageId], [t].[EnglishDisplay], [t].[ISO6393Code], [t].[ScriptDirection], [t].[Status], [t].[MediaType], [t].[Id0], [t].[Id1], [t].[Id2], [t].[c0], [t0].[Id], [t0].[ActualDeliveryDate], [t0].[ActualPublishDate], [t0].[CompanyId], [t0].[CompanyLeadUserId], [t0].[Created], [t0].[EffectiveWordCount], [t0].[LanguageId], [t0].[Name], [t0].[ProjectManagerUserId], [t0].[ProjectPlatformId], [t0].[ProjectedDeliveryDate], [t0].[ProjectedPublishDate], [t0].[QuotedCost], [t0].[SourceWordCount], [t0].[Started], [t0].[Updated]
FROM (
    SELECT TOP(1) [r0].[EnglishLabel], [p].[DisplayName], CASE
        WHEN [p].[LicenseInfo] IS NULL THEN CAST(1 AS bit)
        ELSE CAST(0 AS bit)
    END AS [c], [p].[LicenseInfo], [r].[Id], [r].[ResourceId], [r].[LanguageId], [l].[EnglishDisplay], [l].[ISO6393Code], [l].[ScriptDirection], [r].[Status], [r].[MediaType], CASE
        WHEN EXISTS (
            SELECT 1
            FROM [ResourceContentVersions] AS [r1]
            WHERE [r].[Id] = [r1].[ResourceContentId] AND [r1].[IsPublished] = CAST(1 AS bit)) THEN CAST(1 AS bit)
        ELSE CAST(0 AS bit)
    END AS [c0], [r0].[Id] AS [Id0], [p].[Id] AS [Id1], [l].[Id] AS [Id2]
    FROM [ResourceContents] AS [r]
    INNER JOIN [Resources] AS [r0] ON [r].[ResourceId] = [r0].[Id]
    INNER JOIN [ParentResources] AS [p] ON [r0].[ParentResourceId] = [p].[Id]
    INNER JOIN [Languages] AS [l] ON [r].[LanguageId] = [l].[Id]
    WHERE [r].[Id] = @__request_Id_0
) AS [t]
LEFT JOIN (
    SELECT [t1].[Id], [t1].[ActualDeliveryDate], [t1].[ActualPublishDate], [t1].[CompanyId], [t1].[CompanyLeadUserId], [t1].[Created], [t1].[EffectiveWordCount], [t1].[LanguageId], [t1].[Name], [t1].[ProjectManagerUserId], [t1].[ProjectPlatformId], [t1].[ProjectedDeliveryDate], [t1].[ProjectedPublishDate], [t1].[QuotedCost], [t1].[SourceWordCount], [t1].[Started], [t1].[Updated], [t1].[ResourceContentId]
    FROM (
        SELECT [p1].[Id], [p1].[ActualDeliveryDate], [p1].[ActualPublishDate], [p1].[CompanyId], [p1].[CompanyLeadUserId], [p1].[Created], [p1].[EffectiveWordCount], [p1].[LanguageId], [p1].[Name], [p1].[ProjectManagerUserId], [p1].[ProjectPlatformId], [p1].[ProjectedDeliveryDate], [p1].[ProjectedPublishDate], [p1].[QuotedCost], [p1].[SourceWordCount], [p1].[Started], [p1].[Updated], [p0].[ResourceContentId], ROW_NUMBER() OVER(PARTITION BY [p0].[ResourceContentId] ORDER BY [p0].[ProjectId], [p0].[ResourceContentId], [p1].[Id]) AS [row]
        FROM [ProjectResourceContents] AS [p0]
        INNER JOIN [Projects] AS [p1] ON [p0].[ProjectId] = [p1].[Id]
    ) AS [t1]
    WHERE [t1].[row] <= 1
) AS [t0] ON [t].[Id] = [t0].[ResourceContentId]
ORDER BY [t].[Id], [t].[Id0], [t].[Id1], [t].[Id2];
```
SQL after:
```
DECLARE @__request_Id_0 INT = 6779;
SELECT [t].[EnglishLabel], [t].[DisplayName], [t].[c], [t].[LicenseInfo], [t].[Id], [t].[ResourceId], [t].[LanguageId], [t].[EnglishDisplay], [t].[ISO6393Code], [t].[ScriptDirection], [t].[Status], [t].[MediaType], [t].[Id0], [t].[Id1], [t].[Id2], [t].[c0], [t].[c1], [t0].[Id], [t0].[ActualDeliveryDate], [t0].[ActualPublishDate], [t0].[CompanyId], [t0].[CompanyLeadUserId], [t0].[Created], [t0].[EffectiveWordCount], [t0].[LanguageId], [t0].[Name], [t0].[ProjectManagerUserId], [t0].[ProjectPlatformId], [t0].[ProjectedDeliveryDate], [t0].[ProjectedPublishDate], [t0].[QuotedCost], [t0].[SourceWordCount], [t0].[Started], [t0].[Updated]
FROM (
    SELECT TOP(1) [r0].[EnglishLabel], [p].[DisplayName], CASE
        WHEN [p].[LicenseInfo] IS NULL THEN CAST(1 AS bit)
        ELSE CAST(0 AS bit)
    END AS [c], [p].[LicenseInfo], [r].[Id], [r].[ResourceId], [r].[LanguageId], [l].[EnglishDisplay], [l].[ISO6393Code], [l].[ScriptDirection], [r].[Status], [r].[MediaType], CASE
        WHEN NOT EXISTS (
            SELECT 1
            FROM [ProjectResourceContents] AS [p0]
            WHERE [r].[Id] = [p0].[ResourceContentId] AND [p0].[ResourceContentId] = @__request_Id_0) THEN CAST(1 AS bit)
        ELSE CAST(0 AS bit)
    END AS [c0], CASE
        WHEN EXISTS (
            SELECT 1
            FROM [ResourceContentVersions] AS [r1]
            WHERE [r].[Id] = [r1].[ResourceContentId] AND [r1].[IsPublished] = CAST(1 AS bit)) THEN CAST(1 AS bit)
        ELSE CAST(0 AS bit)
    END AS [c1], [r0].[Id] AS [Id0], [p].[Id] AS [Id1], [l].[Id] AS [Id2]
    FROM [ResourceContents] AS [r]
    INNER JOIN [Resources] AS [r0] ON [r].[ResourceId] = [r0].[Id]
    INNER JOIN [ParentResources] AS [p] ON [r0].[ParentResourceId] = [p].[Id]
    INNER JOIN [Languages] AS [l] ON [r].[LanguageId] = [l].[Id]
    WHERE [r].[Id] = @__request_Id_0
) AS [t]
LEFT JOIN (
    SELECT [t1].[Id], [t1].[ActualDeliveryDate], [t1].[ActualPublishDate], [t1].[CompanyId], [t1].[CompanyLeadUserId], [t1].[Created], [t1].[EffectiveWordCount], [t1].[LanguageId], [t1].[Name], [t1].[ProjectManagerUserId], [t1].[ProjectPlatformId], [t1].[ProjectedDeliveryDate], [t1].[ProjectedPublishDate], [t1].[QuotedCost], [t1].[SourceWordCount], [t1].[Started], [t1].[Updated], [t1].[ResourceContentId]
    FROM (
        SELECT [p2].[Id], [p2].[ActualDeliveryDate], [p2].[ActualPublishDate], [p2].[CompanyId], [p2].[CompanyLeadUserId], [p2].[Created], [p2].[EffectiveWordCount], [p2].[LanguageId], [p2].[Name], [p2].[ProjectManagerUserId], [p2].[ProjectPlatformId], [p2].[ProjectedDeliveryDate], [p2].[ProjectedPublishDate], [p2].[QuotedCost], [p2].[SourceWordCount], [p2].[Started], [p2].[Updated], [p1].[ResourceContentId], ROW_NUMBER() OVER(PARTITION BY [p1].[ResourceContentId] ORDER BY [p1].[ProjectId], [p1].[ResourceContentId], [p2].[Id]) AS [row]
        FROM [ProjectResourceContents] AS [p1]
        INNER JOIN [Projects] AS [p2] ON [p1].[ProjectId] = [p2].[Id]
        WHERE [p1].[ResourceContentId] = @__request_Id_0
    ) AS [t1]
    WHERE [t1].[row] <= 1
) AS [t0] ON [t].[Id] = [t0].[ResourceContentId]
ORDER BY [t].[Id], [t].[Id0], [t].[Id1], [t].[Id2]
```